### ISSUE TITLE
Maybe prefix API endpoints

### ISSUE THREAD

**Issue by cpreston321** [zernonia/simple-log#8]
Since you want integration to happen I'm not for sure the best way to handle it but you could possibly create endpoints that prefix supabase endpoints within the app so then you can create more like a plug and play of different integrations.

Just an Idea! Also maybe include discussions within the repo so I can add some more Idea's instead of adding them to issues :)

**Comment by zernonia** [comment]
Thanks for the idea! But could you give an example of API endpoint prefix?
Currently in we have `/api/v1/supabase/${tokenId}`, do you mean somehting similar?

**Comment by cpreston321** [comment]
Correct! @zernonia but if you want a more plug and play integration system I was thinking more like `/api/v1/${provider}/${token}` then you can route the provider to something more like a plugin system that you can model out in Typescript that you model from. Like how nuxt does `definePlugin()` or `defineNuxt()`. Just an idea for the future!

**Comment by zernonia** [comment]
Good idea! 👍🏻 
But unfortunately @cpreston321 , I'm so tired with all the drama saying this is a copycat product.. and I doesn't wish to continue developing this anymore. Even after I changed the design of the app, there's still people accusing it. 

Nonetheless, I will write out the 'host it yourself' docs 🥲

**Comment by cpreston321** [comment]
Don't let people de-motivate you man. I have made app 10% better and creates competition and more. You got this. Just do you and make a product you want to see come to life you will always have haters....

**Comment by cpreston321** [comment]
I think this product have a lot of potential just needs some work to make it come to life.

**Comment by cpreston321** [comment]
Also this is an Open Source vs non open source. I like the idea of products being open source. You can also just say this more "Open Source Alternative". This gives the developers the power to do anything with it in my opinion.
